pkg/sdn/registry/netnamespace/strategy.go

```diff
@@ -53,7 +53,7 @@ func (sdnStrategy) AllowUnconditionalUpdate() bool {
 
 // ValidateUpdate is the default update validation for a NetNamespace
 func (sdnStrategy) ValidateUpdate(ctx kapi.Context, obj, old runtime.Object) fielderrors.ValidationErrorList {
-	return validation.ValidateNetNamespaceUpdate(obj.(*api.NetNamespace), old.(*api.NetNamespace))
+	return fielderrors.ValidationErrorList{}
```

Comment:
at a minimum, you have to call ValidateObjectMeta during create and ValidateObjectMetaUpdate during update for API objects. Beyond that, if you want to allow mutation of NetID, that's fine.